packages/frontend-v2/utils/theme.ts

```diff
@@ -0,0 +1,83 @@
+import { extendTheme } from "@chakra-ui/react";
```

Comment:
Theme values are currently just whatever i could get from our design system and cindy's notion(for breakpoints). This will evolve eventually.

Reply:
are those colors from the old design system or the new one that linda has?

Reply:
good question! I actually only found 1 design system in the figma([this one](https://www.figma.com/file/CPqFE7T1aJe46zR99n0SPr/GraduateNU-Design-System?node-id=2%3A5)) so the colors are from there, it includes both the blue(from the new designs) and red color(from the current designs). @justineluo and I discussed nailing down a proper new design system in the summer and update the themes to reflect that

Reply:
ahh okay. I think that could be the old design system but I could be wrong. I guess it's okay we can always change those later